yamlkt/src/commonTest/kotlin/net.mamoe.yamlkt/MultilineStringTest.kt

```diff
@@ -0,0 +1,471 @@
+package net.mamoe.yamlkt
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.Serializable
+import kotlin.test.*
+
+@Serializable
+data class TestTwoStringData(val string1: String, val string2: String)
+
+class MultilineStringTest {
+    private val line1 = "This is a multiline string"
+    private val line2 = "This is the second line"
+
+    private lateinit var testStringDataSerializer: KSerializer<DecoderTest.TestStringData>
+    private lateinit var testTwoStringDataSerializer: KSerializer<TestTwoStringData>
+
+    @BeforeTest
+    fun createSerializers() {
+        testStringDataSerializer = DecoderTest.TestStringData.serializer()

```

Comment:
It's better to declare your own TestData class in this test. So `DecoderTest` can be changed in the future without any concern.

Reply:
Oh yeah good call, done in 531d708